README.md

```diff
@@ -0,0 +1,7 @@
+# Appcircle Tuist Commands 
+
+This step runs specific [Tuist](https://tuist.io/) commands.
```

Comment:
You can leave it like this, but you can also update it as follows:
```suggestion
This step runs specific [Tuist](https://tuist.io/) commands that you provide as input. For example; `tuist build` or `tuist install`.
```

Reply:
I thought about it too, but I didn't want it to be dublicate because I specified it in the required input section below.

Reply:
> I thought about it too, but I didn't want it to be dublicate because I specified it in the required input section below.

right, but it can also:
```suggestion
This step runs specific [Tuist](https://tuist.io/) commands that you provide as input. 
```

Reply:
This is more make sense. I applied directly the changes, thx. :)